config.json

```diff
@@ -2448,6 +2448,25 @@
           "multiple-clause-matching"
         ],
         "difficulty": 5
+      },
+      {
+        "slug": "affine-cipher",
+        "name": "Affine Cipher",
+        "uuid": "18d12bac-ead0-456c-bcef-ccb2fd06fe72",
+        "prerequisites": [
+          "pattern-matching",
+          "atoms",
+          "enum",
+          "integers",
+          "if",
+          "strings",
+          "tuples"
+        ],
+        "practices": [
+          "enum",
```

Comment:
Sorry, I ran the checks on a slightly older branch that didn't have the concept tracking. 
I'll remove `enum`. 
Maybe I could add `pipe-operator` instead? `encode` is just one clean pipe.